editors/code/src/highlighting.ts

```diff
@@ -54,10 +47,10 @@ export function activateHighlighting(ctx: Ctx) {
 }
 
 // Based on this HSL-based color generator: https://gist.github.com/bendc/76c48ce53299e6078a76
-function fancify(seed: string, shade: 'light' | 'dark') {
+function fancify(seed: string, shade: 'light' | 'dark'): string {
     const random = randomU32Numbers(hashString(seed));
-    const randomInt = (min: number, max: number) => {
-        return Math.abs(random()) % (max - min + 1) + min;
+    const randomInt = (min: number, max: number): number => {
```

Comment:
I do like TypeScript for not forcing you to write types here and there so that the code is very concise but it still provides you with the strong static typing guarantees.
I'd actually even remove all the return type declarations if I had enough impudence, but I'd suggest at least disabling the rule for arrow functions. @matklad ?

Reply:
Okay, I can disable that. I agree it can be a bit much.